ethereum/contracts/TablelandRigPilots.sol

```diff
@@ -279,6 +279,50 @@ contract TablelandRigPilots is
         emit Training(tokenId);
     }
 
+    /**
+     * @dev See {ITablelandRigPilots-pilotRig}.
+     */
+    function pilotRig(address sender, uint256 tokenId) external onlyParent {
```

Comment:
An overloaded `pilotRig` doesn't take any pilot contract nor ID values and will assume a `0x0` address with pilot ID hardcoded as `2`.

Only a parked Rig can have a trainer pilot set with this method. It will insert these values into the sessions table, which also helps ensure training-related sessions differ from piloting with a stock / trainer pilot.